Silicon/QemuSocPkg/Include/PlatformBase.h

```diff
@@ -7,6 +7,11 @@
 
 #ifndef _PLATFORM_BASE_H_
 
+#define PCI_TO_CF8_ADDRESS(A) \
+  ((UINT32) ((((A) >> 4) & 0x00ffff00) | ((A) & 0xfc) | 0x80000000))
+
+#define  INTEL_X58_ICH10_DEVICE_ID         0x3400  // Host Bridge DID for Simics QSP
+#define  PLATFORM_ID_QSP_SIMICS            0x2

```

Comment:
Better to put it in
Platform\xxxxBoardPkg\Include\ConfigDataDefs.h

Reply:
Noted, because this is board specific and its consistent with other platforms. done.